cmd/client/alert.go

```diff
@@ -71,7 +71,7 @@ var AlertCreate = &cobra.Command{
 			exitOnError(err)
 		}
 
-		if err := client.Create("alert", &alert); err != nil {
+		if err := client.Create("alert", &alert, "json"); err != nil {
```

Comment:
may define "json"/"yaml" as a const type in common/types.go  (a kind like SortOrder)
type APIMessageFormat int

Reply:
@nplanel Done, Thanks!